rustls/src/server/tls13.rs

```diff
@@ -504,6 +586,57 @@ mod client_hello {
         common.send_msg(m, false);
     }
 
+    fn decide_if_early_data_allowed(
```

Comment:
Style nit: note that the argument order here is sort of reversed from other nearby functions, which sort roughly from most changing/unique to least changing/unique.